carsus/io/output/base.py

```diff
@@ -1007,10 +1014,16 @@ def to_hdf(self, fname):
                   self.macro_atom_references_prepared)
             
             if hasattr(self, 'collisions'):
-                if self.collisions_metadata.name == "chianti":
```

Comment:
This if-else block didn't fail because all the `pandas.Series` objects have a `name` attribute. In this case `name` was empty and fall back to the `else` statement.

The correct name was `self.collisions_metadata.dataset` as it was defined in the first lines of this module and pointed out by @laureanomartinez.